app/_src/gateway/breaking-changes/310x.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Kong Gateway 3.10.x breaking changes
+content_type: reference
+book: breaking-changes
+chapter: 12
+---
+
+Before upgrading, review any configuration or breaking changes in this version and prior versions that
+affect your current installation.
+
+You may need to adopt different [upgrade paths](/gateway/{{page.release}}/upgrade/#guaranteed-upgrade-paths) depending on your 
+deployment methods, set of features in use, or custom plugins, for example.
+
+Review the [changelog](/gateway/changelog/#31000) for all the changes in this release.
+
+## Breaking changes and deprecations
+
+### AI plugins: metrics key
+
+The serialized log key of AI metrics has changed from `ai.ai-proxy` to `ai.proxy` to avoid conflicts with metrics generated from plugins other than AI Proxy and AI Proxy Advanced. 
+If you are using any [logging plugins](/hub/?category=logging) to log AI metrics (for example, File Log, HTTP Log, and so on), 
+you will have to update metrics pipeline configurations to reflect this change.
+
+### AI plugins: deprecated settings
+
+The following settings have been deprecated in all [AI plugins](/hub/?category=ai), and will be removed in a future release.
+Use the following replacement settings instead:
+
+| Deprecated setting | New setting | 
+|--------------------|-------------|
+| `preserve` mode in `config.route_type` | `config.llm_format` |
+| `config.model.options.upstream_path`   | `config.model.options.upstream_url` |
+
+### AI Rate Limiting Advanced plugin: multiple providers and limits
+
+The plugin's `config.llm_providers.window_size` and `config.llm_providers.limit` parameters now require an array of numbers instead of a single number. 
+If you configured the plugin before 3.10 and upgrade to 3.10 using `kong migrations`, it will be automatically migrated to use an array.
```

Comment:
```suggestion
If you configured the plugin before 3.10 and [upgrade to 3.10 using `kong migrations`](/gateway/{{page.release}}/upgrade/), it will be automatically migrated to use an array.
```